### DISCUSSION TITLE
What to use instead of mock_secretsmanager

### DISCUSSION THREAD

**Discussion by jgarte** [spulec/moto#8309] [Q&A]
Hi,

What should I be using now instead of `mock_secretsmanager` context manager?

**Comment by bblommers** [comment]
Hi @jgarte, you can use `mock_aws` instead. This is a replacement for all previous decorators, i.e. `mock_s3`, `mock_dynamodb`, etc.

```python
from moto import mock_aws

@mock_aws
def test():
    ...
```

See the docs for all breaking changes: 
https://docs.getmoto.org/en/latest/docs/releases.html#breaking-changes
